### PR TITLE
upgraded azure/serilog dependencies

### PR DESCRIPTION
**Pull request by vip32**
Hi,

Due to issue #42 I made this PR.

- document db nuget updated to 1.11.1 (most recent)
- serilog nuget updated to 2.3.0  (most recent)

Please do find some time to pull this in. I marked the nuget as a beta prerelease (so it can be tried before releasing). thx

### CODE REVIEW

**Comment by nblumhardt** [comment]
@saleem-mirza I just noticed the default branch of this repo is `master` - in general the other Serilog repositories use `dev` as the default branch so that contributions target it automatically (this makes it less likely someone will waste time forking and working against master when dev has already moved forward). I can make the branch change here but thought I'd drop you a line to avoid surprises. All systems "go"?

**Comment by saleem-mirza** [comment]
@nblumhardt yes, go by all means. Thank you for correction.

**Comment by nblumhardt** [comment]
Done! Also re-targeted the PR.

File: src/Serilog.Sinks.AzureDocumentDb/project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Serilog.Sinks.AzureDocumentDB",
-    "version": "3.5.17-*",
+    "version": "3.5.18-beta",
```
Comment (nblumhardt) [inline src/Serilog.Sinks.AzureDocumentDb/project.json]: This isn't necessary - on the `dev` branch, the version suffix will be added automatically in place of the `*`.

File: src/Serilog.Sinks.AzureDocumentDb/project.json [hunk 2]
```diff
@@ -23,19 +23,19 @@
         "embed": [ "/**/bulkImport.js" ],
         "define": []
     },
-    "dependencies": {
-        "Serilog": "2.2.1"
-    },
+  "dependencies": {
```
Comment (nblumhardt) [inline src/Serilog.Sinks.AzureDocumentDb/project.json]: Whitespace formatting here?

Comment (vip32) [inline src/Serilog.Sinks.AzureDocumentDb/project.json]: yes

File: src/Serilog.Sinks.AzureDocumentDb/project.json (hunk 1 above)
Comment (vip32) [inline src/Serilog.Sinks.AzureDocumentDb/project.json]: ah ok

**Comment by nblumhardt** [comment]
(@vip32 just a heads-up, you can respond to comments/make modifications by pushing more commits to the branch you're making the PR from. Cheers!)

**Comment by vip32** [comment]
not sure, but should i fix something in the PR?

**Comment by nblumhardt** [comment]
@vip32 Hi, yes, the whitespace formatting and version numbering mentioned above 👍 thanks!

**Comment by vip32** [comment]
ok, done with the adjustments

**Comment by philcarbone** [comment]
Thanks for this fix! We are waiting on it :)

**Comment by saleem-mirza** [comment]
Looks good 👍, thanks.

**Comment by vip32** [comment]
thx for merging, when will this new release be on nuget? what is your schedule for that?

**Comment by saleem-mirza** [comment]
@vip32  pre-release nuget package has been published. Once everything goes well with pre-release, we'll merge it into release version. see https://www.nuget.org/packages/Serilog.Sinks.AzureDocumentDb/3.5.18-dev-00119

**Comment by vip32** [comment]
great, i just pulled it into my project and now serilog logs nicely to documentdb (and using Microsoft.Azure.DocumentDB 1.11.1)

**Comment by philcarbone** [comment]
looks good. works as expected

**Pull request merged**